### PR TITLE
Wrap: Clarify that passing a collection uses the first element

### PR DESCRIPTION
**Pull request by arthurvr**
Fixes gh-698.

### CODE REVIEW

**Comment by dmethvin** [comment]
LGTM thanks!

File: entries/wrap.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       <type name="htmlString"/>
       <type name="Element"/>
       <type name="jQuery"/>
-      <desc>A selector, element, HTML string, or jQuery object specifying the structure to wrap around the matched elements.</desc>
+      <desc>A selector, element, HTML string, or jQuery object specifying the structure to wrap around the matched elements. When you pass a jQuery collection containing more than one element the first element will be used.</desc>
```
Comment (AurelioDeRosa) [inline entries/wrap.xml]: This happens also when you provide a selector matching more than one element @arthurvr.

**Comment by AurelioDeRosa** [comment]
Have you read my comment @arthurvr?

**Comment by arthurvr** [comment]
> Have you read my comment @arthurvr?

Yes, but I read it too late.

**Comment by arthurvr** [comment]
Will do a PR with a fix up.

**Pull request closed without merging**